packages/enzyme/src/Utils.js

```diff
@@ -320,3 +320,49 @@ export function spyMethod(instance, methodName) {
     },
   };
 }
+
+export function validEmptyValues(renderedValue) {
+  return renderedValue === null || renderedValue === false;
+}
+
+export function renderedDive(nodes) {
+  let isEmptyRender = true;
+  const renderedKey = 'rendered';
+
+  function dive(renderedNodes) {
+    const isList = renderedNodes && renderedNodes.length > 0 && Array.isArray(renderedNodes);
+    if (isList) {
+      return renderedNodes.some((n) => {
+        if (n) {
+          if ((has(n, renderedKey))) {
+            dive(n.rendered);
```

Comment:
should this be `n[renderedKey]`?

Reply:
or maybe we don't need `renderedKey` at all, and we can just hardcode `.rendered`